tests/pages/api/timeRestrictions.test.ts

```diff
@@ -0,0 +1,87 @@
+import { timeRestrictionsErrorId } from '../../../src/pages/timeRestrictions';
+import * as utils from '../../../src/pages/api/apiUtils/index';
+import timeRestrictions from '../../../src/pages/api/timeRestrictions';
+import { getMockRequestAndResponse } from '../../testData/mockData';
+import { isSessionValid } from '../../../src/pages/api/apiUtils/validator';
+import * as sessionUtils from '../../../src/utils/sessions';
+import { TIME_RESTRICTIONS_ATTRIBUTE } from '../../../src/constants';
+
+beforeEach(() => {
+    (isSessionValid as {}) = jest.fn().mockReturnValue(true);
+});
+
+afterEach(() => {
+    jest.resetAllMocks();
+});
+
```

Comment:
Should these `beforeEach` and `afterEach` functions be inside the first 'describe'?

Reply:
As theres only one describe, works either way.